react-client/src/components/BookCard.jsx

```diff
@@ -73,7 +76,12 @@ class BookCard extends React.Component {
   }
 
   goToBook() {
-    this.props.changeView('Book', this.state.book);
+    this.props.getProReviews(this.state.isbn, (response) => {
```

Comment:
We may want to move this into the book page onMount. It might slow down the click response and cause a user to click on multiple things.